cadccutout/cadccutout/tests/test_core.py

```diff
@@ -127,3 +138,28 @@ def test_write_stream():
     test_subject.write(b'You have been recruied by the Star League to defend \
             the frontier against Xur and the Kodhan Armada.')
     assert test_subject.tell() == 111, 'Message written.'
+
+
+def test_construct():
+    test_subject = OpenCADCCutout()
+
+    try:
```

Comment:
Use the pytest construct:
with pytest.raises(<ExpectedError>) as e:
    <test_something>